fordel/src/main/java/no/nav/ung/fordel/kafka/DefaultKafkaSettings.java

```diff
@@ -0,0 +1,195 @@
+    }
+
+    @Override
+    public String getSchemaRegistryUrl() {
+        return schemaRegistryUrl;
+    }
+
+    @Override
+    public Map<String, String> schemaRegistryProps() {
+        var props = new HashMap<String, String>();
+        if (schemaRegistryUrl != null) {
+            props.put("schema.registry.url", schemaRegistryUrl);
+        }
+        return props;
+    }
+
+    private String getUsername() {
+        return username;
+    }
+
+    private String getPassword() {
+        return password;
+    }
+
+    private boolean harSattBrukernavn() {
+        return username != null && !username.isEmpty();
+    }
+
+    private String getApplicationId() {
+        return applicationId;
+    }
+
+    private boolean harSattTrustStore() {
+        return trustStorePath != null && !trustStorePath.isEmpty()
+                && trustStorePassword != null && !trustStorePassword.isEmpty();
+    }
+
+    private String getTrustStorePath() {
+        return trustStorePath;
+    }
+
+    private String getTrustStorePassword() {
+        return trustStorePassword;
+    }
+
+    @Override
+    public List<String> getDeactivatedTopics() {
+        return deactivatedTopics;
+    }
+
+    @Override
+    public Properties toStreamPropertiesWith(String clientId, Serde<?> keySerde, Serde<?> valueSerde) {
+        final Properties props = baseProperties();
+
+        /*
+         * Application ID må være unik per strøm for å unngå en feilsituasjon der
+         * man enkelte ganger får feil partition (dvs partitions fra annen topic
+         * enn den man skal ha).
+         */
+        props.put(StreamsConfig.APPLICATION_ID_CONFIG, getApplicationId() + "-" + clientId);
+        props.put(StreamsConfig.CLIENT_ID_CONFIG, clientId);
+        if (readAllRecords) {
+            props.put("auto.offset.reset", "earliest");
+        } else {
+            props.put("auto.offset.reset", "none");
+        }
+
+        // Serde
+        props.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, keySerde.getClass());
+        props.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, valueSerde.getClass());
+        props.put(StreamsConfig.DEFAULT_DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG, LogAndFailExceptionHandler.class);
+
+        return props;
+    }
+
+    @Override
+    public Properties toProducerPropertiesWith(String navn) {
+        var producerClientId = String.format("producer-%s-%s", navn, clientId());
+        final Properties props = baseProperties();
+        props.put(ProducerConfig.CLIENT_ID_CONFIG, producerClientId);
+        props.put(ProducerConfig.ACKS_CONFIG, "1");
+        props.put(ProducerConfig.LINGER_MS_CONFIG, "0");
+        props.put(ProducerConfig.MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION, "1");
+        return props;
+    }
+
+    @Override
+    public Properties toStreamPropertiesWith(String navn) {
+        final Properties props = baseProperties();
+        var streamId = String.format("stream-%s-k9-fordel", navn);
```

Comment:
bør antagelig endre navn her for at både k9-fordel og ung-ska skal få lese alle meldinger